test/distributed/org/apache/cassandra/distributed/test/TestBaseImpl.java

```diff
@@ -43,4 +51,29 @@ public static void beforeClass() throws Throwable {
         // fact that this code is going to work accross _all_ versions.
         return Cluster.build();
     }
+
+    public static Object list(Object...values)
+    {
+        return CQLTester.list(values);
+    }
+
+    public static Object set(Object...values)
+    {
+        return CQLTester.set(values);
+    }
+
+    public static Object map(Object...values)
+    {
+        return CQLTester.map(values);
```

Comment:
nit: Not related to this patch, but `map()` has a couple type warnings. We could change line 1782 to `Map<Object, Object> m = new LinkedHashMap<>(size);`

Reply:
Done in both `TestBaseImpl.map` and the original `CQLTester#map`.